packages/riverpod_generator/lib/src/riverpod_generator.dart

```diff
@@ -20,6 +20,28 @@ String providerDocFor(Element element) {
       : '${element.documentationComment}\n///\n/// Copied from [${element.name}].';
 }
 
+final metaAnnotationsTypeChecker = TypeChecker.any([
+  const TypeChecker.fromRuntime(Deprecated),
+  TypeChecker.fromRuntime(experimental.runtimeType),
```

Comment:
The classes of annotations from meta are all private.
```dart
const _Experimental experimental = _Experimental();

class _Experimental {
  const _Experimental();
}
```
Is there a better way to get it's type?

Reply:
Don't use TypeChecker to begin with.

Meta annotations are official. There's straight up a `isExperimental`/`isDeprecated` getter in the analyzer

Reply:
Actually isExperimental might be the exception. It seems to be missing. We probably don't care too much about it anyway. I doubt people use it